templates/solutions/infrastructure.html

```diff
@@ -0,0 +1,543 @@
+                <div class="p-section--shallow">
+                    <h3 class="p-heading--2">Manage your cloud</h3>
+                </div>
+                <div class="p-card__content p-section--shallow">
+                    <p class="p-section--shallow">You manage your cloud yourself. We can assist you with the design,
+                        help you make the right hardware choices, deploy it for you, provide full commercial support and
+                        train your team.</p>
+                    <hr />
+                    <a class="p-section--shallow" style="float: right;" href="/contact-us"
+                        aria-controls="infrastructure-modal">Contact us to discuss your needs&nbsp;&rsaquo;</a>
+                </div>
+            </div>
+        </div>
+        <div class="col">
+            <hr class="p-rule--highlight" />
+            <div class="p-card">
+                <div class="p-section--shallow">
+                    <h3 class="p-heading--2">Let us manage it</h3>
+                </div>
+                <div class="p-card__content p-section--shallow">
+                    <p class="p-section--shallow">We deploy and manage your cloud 24/7. This includes typical daily maintenance tasks, incident and problem resolution, recurring upgrades and more. You benefit from a public cloud-like experience.</p>
+                    <hr />
+                    <a class="p-section--shallow" style="float: right;" href="https://ubuntu.com/managed">Browse managed IT services&nbsp;&rsaquo;</a>
+                </div>
+            </div>
+        </div>
+    </div>
+</section>
+
+<section class="p-section--deep">
+    <div class="row--50-50">
+        <hr class="p-rule" />
+        <div class="col">
+            <h2>Take the next step in your journey</h2>
+        </div>
+        <div class="col">
+            <ul class="p-list--divided">
+                <li class="p-list__item">
+                    <a href="https://ubuntu.com/engage/multi-cloud-business-guide">Download our business guide to multi-cloud</a>
+                </li>
+                <li class="p-list__item">
+                    <a href="https://ubuntu.com/engage/vmware-migration-scenarios">Migrate from VMware to open source</a>
+                </li>
+                <li class="p-list__item">
+                   <a href="https://ubuntu.com/engage/software-defined-storage-for-enterprises">Software-defined storage for enterprises</a>
+                </li>
+                <li class="p-list__item">
+                    Send us a message to discuss your project.
+                </li>
+            </ul>
+            <a class="p-button--positive" href="/contact-us" aria-controls="infrastructure-modal">Contact us</a>
+        </div>
+    </div>
+</section>
+
+<script defer src="{{ versioned_static('js/modals.js') }}"></script>
+
+{% with %}
+{% set returnURL = "https://canonical-com-1184.demos.haus/solutions/infrastructure#infrastructure-form-success" %}
```

Comment:
This does not work with relative path (same as in the other form I was doing). It will be updated after testing as done in here: https://github.com/canonical/canonical.com/pull/1173

Reply:
ok let's make sure you don't forget it